.eslintrc-node.json

```diff
@@ -1,8 +1,14 @@
 {
   "extends": "./.eslintrc-base.json",
-
   "env": {
     "browser": false,
     "node": true
-  }
+  },
+  "parserOptions": {
+    // Required for certain syntax usages
+    "ecmaVersion": 6
```

Comment:
Let's change it to `2017` as Node 8 already supports it: https://kangax.github.io/compat-table/es2016plus/

Reply:
Done